packages/core/src/endowmentsToolkit.js

```diff
@@ -114,6 +121,14 @@ function endowmentsToolkit({
           }
           allowedWriteFields.add(path)
           whitelistedReads.push(path)
+          if (packagePolicyValue === 'write+define') {
```

Comment:
I guess I don't love `write+define`. at first glance, this seems like _multiple_ permissions, and multiple permissions want to be expressed as an object or array.

what of `overwrite`?  or is this _specifically_ for `Object.defineProperty()`?

Reply:
I was trying to come up with a convention that would avoid `[].includes()` being the way we handle writeable stuff. 
It's literally the same as `write` but with the added allowance to use a monkey-patch on defineProperty